### DISCUSSION TITLE
why not microfederation?

### DISCUSSION THREAD

**Discussion by SoniEx2** [mastodon/mastodon#21242] [General]
currently mastodon takes a global approach. why not take an approach of small groups of microfederated instances tied together by a process/protocol that makes it easy to join new ones?

you use your existing microfederated instance to find more microfederated instances, and as you go through them, your knowledge of microfederated instances expand.

this would be great, it'd encourage having an account (ideally the same account) on multiple instances, it'd completely eliminate the issues inherent to defederation (mostly the issue of cutting ppl off from their friends, which even mastodon.social is guilty of), it'd also push for inclusion of sex workers and neurodivergent folks into more communities.

it also solves the scaling problem. you don't need to scale nearly as much if you only peer with another 5 to 10 instances. that would allow making the server software much more lightweight too, thus leading to... indeed, more instances!

just, what if php-like forums with lightweight opt-in federation/peering.

**Comment by ineffyble** [comment]
Webrings for the modern age!

**Reply by ineffyble** [reply]
Tootrings? 🤔

**Reply by SoniEx2** [reply]
toot servers we guess (a la discord servers). toot guilds?

too bad "toot" got deprecated in mastodon 4.0 >.<
